bin/sl-deploy.js

```diff
@@ -81,6 +87,9 @@ if (!serviceName) {
   process.exit(1);
 }
 
+clusterSize = (isNaN(clusterSize) || clusterSize < 1) ?
```

Comment:
cluster sizes < 1 are valid,  I set the size to 0 all the time for testing. Negative sizes are also valid (they mean don't restart on failure, arc uses a size of `-1` for local deploys). It would also be nice to support the string `/cpus?/i`, which is a valid size.